CONTRIBUTING.md

```diff
@@ -61,31 +61,91 @@ You can also clean up generated files:
 
 `yarn clean`
 
+## Releasing packages
 
-## Pull Requests, versioning and releasing
+Automated releases to the npm public registry are triggered for all pull requests containing modifications to one or more npm packages (found in the `/packages/` directory). The information required to determine the version update for each release is taken from the title and content of the pull request.
 
-To deploy Kaizen Component Library changes, your **Pull Request (PR) title must follow the [Conventional Commits format](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)** and “Semantic Versioning” (SemVer):
+### Release workflow
 
-> Given a version number MAJOR.MINOR.PATCH, increment the:
-    MAJOR version when you make incompatible API changes,
-    MINOR version when you add functionality in a backwards compatible manner, and
-    PATCH version when you make backwards compatible bug fixes.
+To release a new package version, create a pull request which **modifies ONLY the package you wish to release**, and satisfies **ONE** of the following:
+
+- If the branch contains multiple commits, the pull request must have a [conventional title](#conventional-commit)
+- If the branch contains only a single commit...
+  - ... the commit itself must have a [conventional commit message](#conventional-commit), and...
+  - ... (for consistency) the PR title must match that commit message
+
+Once that pull request is merged into master, an automated release will be triggered, and the newly published package version will be available on the npm public registry.
+
+### Semantic Versioning
+
+All npm packages follow strict semantic versioning (or _semver_). Semantic versioning gives specific meaning to changes in version numbers, as follows:
```

Comment:
Great clarification of where the behaviour comes from (all npm packages follow semver) 👍